### DISCUSSION TITLE
Missed validate() method in documentation

### DISCUSSION THREAD

**Discussion by iAdil** [livewire/livewire#6433] [7. Bugs (archived)]
### Livewire version

3.0@beta

### Which PHP version are you using?

PHP 8.1

### Steps To Reproduce

Documentation is a little bit confused, for example in [Real-time form saving](https://livewire.laravel.com/docs/forms#real-time-form-saving)

`Real-time form saving
If you want to automatically save a form as the user fills it out rather than wait until the user clicks "submit", you can do so using Livewire's updated() hook:
`

`Additionally, because we have the #[Rule] attributes attached to those properties, the validation rules will be run before the property is updated and the updated() hook is called.`

### Additional information

**The updated method will execute anyway until we add the line:**
` $this->validate();`

Is it a code bug or a documentation mistake?
### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by i2iMatt** [comment]
Can somebody action this, how has this been ignored for 2 years.
